doc/us-en/setup/Quick-Install.md

```diff
@@ -0,0 +1,69 @@
+
+**For more details, please see the [official documentation](https://docs.docker.com/install/).**
+
+## Install OpenEdge
+
+The rpm and deb packages will be released accordingly when OpenEdge releases a new version. Users can install OpenEdge to the device through package manager with following command:
+
+```shell
+curl -sSL http://download.openedge.tech/install.sh | sudo -E bash -
+```
+
+OpenEdge will be installed to the `/usr/local` directory after the execution is complete.
+
+The latest version of OpenEdge uses `Systemd` daemon, and users can launch OpenEdge with the following command:
+
+```shell
+sudo systemctl start openedge
+```
+
+## Import the default configuration (optional)
+
+As an edge computing platform, OpenEdge provides basic functional modules, such as hub and function-manager modules, in addition to the underlying service management capabilities. Users can make `openedge` main program to load corresponding modules and set running parameters of the modules themselves with editing the configuration files. For an introduction to each module, refer to [Configuration Interpretation](../tutorials/Config-interpretation.md) for further information.
+
+OpenEdge officially provides a default configuration which can be imported using following command:
+
+```shell
+curl -sSL http://download.openedge.tech/install_with_docker_example.sh | sudo -E bash -
+```
+
+The default configuration is for learning and testing purposes only. Users should perform on-demand configuration according to actual working scenarios. For details, refer to [Configuration Interpretation](../tutorials/Config-interpretation.md) for further understanding.
+
+There is no need to import any configuration files if no module need to launch.
+
+## Verify successful installation
+
+After installing through package manager, users can verify that OpenEdge is successfully installed by following these steps:
+
+- executing the command `sudo systemctl status openedge` to check whether `openedge` is running, as shown below. Otherwise, `openedge` fails to start.
+
+![OpenEdge](../../images/setup/openedge-systemctl-status.png)
+
+- executing the command `docker stats` to view the running status of docker containers. Since the main program `openedge` will first pull required images from mirror repository, users needs to wait 2~5 minutes to execute this command. Take the default configurations in above step for example, the running status of containers are as shown following after pulled. If some containers are missing, it says they failed to start.
+
+![docker stats](../../images/setup/docker-stats.png)
+
+- Under the condition of two above failures, you need to view the log to check more of main program. Log files are stored in `/usr/local/var/log/openedge/openedge.log` by default. For errors in the log, users can refer to [FAQ](../FAQ.md). If necessary, just [Submit an issue](https://github.com/baidu/openedge/issues) for help.
```

Comment:
Under the condition of two above failures, you need to view the log of main program. And the log file which is stored in `/usr/local/var/log/openedge/openedge.log` by default. Once found errors in the log file, users can refer to [FAQ](../FAQ.md). If necessary, just [Submit an issue](https://github.com/baidu/openedge/issues) for help.